external/Eigen/src/Core/arch/NEON/Complex.h

```diff
@@ -18,7 +18,7 @@ namespace internal {
 inline uint32x4_t p4ui_CONJ_XOR()
 {
 // See bug 1325, clang fails to call vld1q_u64.
-#if EIGEN_COMP_CLANG || EIGEN_COMP_CASTXML
+#if EIGEN_COMP_CLANG || EIGEN_COMP_CASTXML || EIGEN_COMP_CRAY || defined(__CLANG_FUJITSU)
```

Comment:
are these bugs that we should report to Eigen?

Reply:
I have been reporting a bunch of issues and PRs to the Eigen developers in the last couple of weeks. After the switch to C++14 (v3.4.x) the library is undergoing heavy redesign. I suggest we update to v3.4.1 once it's out.